app/addons/documents/shared-resources.js

```diff
@@ -36,9 +36,9 @@ define([
     initialize: function (_attrs, options) {
       if (this.collection && this.collection.database) {
         this.database = this.collection.database;
-      } else if (options.database) {
-        this.database = options.database;
+        return;
       }
+      this.database = options.database;
```

Comment:
The model can't work in any way without setting a database (e.g. see `url` method)